whelk-core/src/main/groovy/whelk/component/PostgreSQLComponent.groovy

```diff
@@ -309,11 +316,26 @@ class PostgreSQLComponent {
             AND l.data #>> '{@graph,1,heldBy,@id}' = ANY(?);
             """.stripIndent()
 
+    private static final String SPARQL_QUEUE_ADD = "INSERT INTO lddb__sparql_q (id) VALUES (?)"
+
+    private static final String SPARQL_QUEUE_REMOVE = """
+            DELETE FROM lddb__sparql_q
+            WHERE pk = (
+              SELECT pk
+              FROM lddb__sparql_q
+              ORDER BY pk
```

Comment:
Maybe add 'ASC' explicitly? I can never remeber the deafult sort order and had to look it up.